SayTheirNamesTests/ServiceTests/Requestors/PeopleNetworkRequestorTests.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PeopleNetworkRequestorTests.swift
+//  SayTheirNamesTests
+//
+//  Created by Oliver Binns on 05/06/2020.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Mocker
+import XCTest
+@testable import Say_Their_Names
+
+extension NetworkRequestorTests {
+    func test_fetchPeople_makesRequest() {
+        guard let apiEndpoint = URL(string: "https://saytheirnames.dev/api/people") else {
```

Comment:
Can we not use `PersonEnvironment.urlString` here? So our URLs are kept in one place.

Reply:
Done ✅